client/src/Client/pages/FirstSignUp/FirstSignUp.tsx

```diff
@@ -46,7 +46,7 @@ const FirstSignUp = () => {
       <img src={logo} alt="logo" className="mb-2.5" />
       <h1 className="text-center">
         שירות הסעות התנדבותי
-        {serverEnvSettings?.allowGuestRideMode ? '' : 'למבקרים בבית חולים'}
+        {serverEnvSettings?.allowGuestRideMode ? '' : ' למבקרים בבית חולים ' }
       </h1>
```

Comment:
?why were spaces added

Reply:
In the old version it was necessary (the text would connect...), in the current version it is not necessary.